base/field_abilities/PoundFieldPsynergy.ts

```diff
@@ -0,0 +1,266 @@
+            );
+            this.hand_hit_ground();
+        });
+        timer.start();
+    }
+
+    hand_hit_ground() {
+        let timer = this.game.time.create(true);
+        timer.add(200, () => {
+            this.hand_translate_x = this.hand_sprite.centerX;
+            this.hand_translate_y = this.hand_sprite.centerY;
+            this.move_hand_by_dir(0, -10);
+            this.hand_tween(this.hand_translate_x, this.hand_translate_y, 400, () => {
+                let timer2 = this.game.time.create(true);
+                timer2.add(200, () => {
+                    if (this.cast_direction === directions.up || this.cast_direction === directions.down) {
+                        this.move_hand_by_dir(0, this.controllable_char.sprite.height + 12);
+                    } else {
+                        this.move_hand_by_dir(0, this.controllable_char.sprite.height);
+                    }
+                    if (this.target_found) {
+                        this.game.add
+                            .tween(this.target_object.sprite.scale)
+                            .to({y: 0.1}, 150, Phaser.Easing.Linear.None, true);
+                    }
+                    this.hand_tween(
+                        this.hand_translate_x,
+                        this.hand_translate_y,
+                        150,
+                        this.finish_hand_movement.bind(this)
+                    );
+                });
+                timer2.start();
+            });
+        });
+        timer.start();
+    }
+
+    finish_hand_movement() {
+        this.data.audio.play_se("pound1/se1");
+        if (this.target_found) {
+            this.target_object.sprite.scale.y = 1;
+            const key_name = this.target_object.sprite_info.key_name;
+            const anim_key = this.target_object.sprite_info.getAnimationKey(key_name, "down");
+            this.target_object.sprite.animations.play(anim_key);
+            this.target_object.get_events().forEach((event: JumpEvent) => {
+                if (event.is_set) {
+                    event.deactivate();
+                    event.is_set = false;
+                } else {
+                    event.activate();
+                    event.is_set = true;
+                    JumpEvent.active_jump_surroundings(
+                        this.data,
+                        get_surroundings(event.x, event.y, false, 2),
+                        event.collision_layer_shift_from_source + this.target_object.base_collision_layer
+                    );
+                }
+            });
+            let shadow_index = this.data.npc_group.getChildIndex(this.data.hero.sprite) - 1;
+            this.data.npc_group.setChildIndex(this.target_object.sprite, shadow_index);
+            this.target_object.sprite.send_to_back = true;
+            this.data.npc_group.sendToBack(this.target_object.sprite);
+            this.target_object.sprite.body.removeCollisionGroup(this.data.collision.hero_collision_group, true);
```

Comment:
This is not going to work if the map changes its collision structure (change of collision layer). You can see this by using pound, climb the house and then back, you'll see that the hero will collide with the pound object. You can just delete the collision body to solve this.